_includes/footer.html

```diff
@@ -0,0 +1,63 @@
+
+        <footer class="page-footer">
+          <div class="container">
+            <div class="row">
+              <div class="col l6 s12">
+                <h5 class="white-text">viperdev.io</h5>
+                <p class="grey-text text-lighten-4">Improving code quality with awesome workshops.</p>
```

Comment:
that supposed to be the oneline description? Under the heading on the top you hae `THE LEADING EDGE WAY TO IMPROVE CODE QUALITY` which isn't completely convincing to me either but better and it should be consistent